### PR TITLE
Close #4068 patch update using the notifications api [es]

### LINKED ISSUE #4068: 24. /es/docs/web/api/notifications_api/using_the_notifications_api - [en-US]

### PR DESCRIPTION
**Pull request by Graywolf9**
- [x] Remove the macro `page`
- [x] Update content
- [x] Transform to `md`

### CODE REVIEW

File: files/es/web/api/notifications_api/using_the_notifications_api/index.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+### Resultado
+
+{{ EmbedLiveSample('Tag_example', '100%', 30) }}
+
+## Especificaciones:
+
+{{Specifications("api.Notification")}}
+
+## Compatibilidad con Navegadores
```
Comment (Graywolf9) [inline files/es/web/api/notifications_api/using_the_notifications_api/index.md:262]:
```suggestion
## Compatibilidad con navegadores
```

File: files/es/web/api/notifications_api/using_the_notifications_api/index.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
+---
+title: Usando la API de Notificaciones
+slug: Web/API/Notifications_API/Using_the_Notifications_API
+translation_of: Web/API/Notifications_API/Using_the_Notifications_API
+original_slug: Web/API/Notifications_API/Usando_la_API_de_Notificaciones
+---
+
```
Comment (Vallejoanderson) [inline files/es/web/api/notifications_api/using_the_notifications_api/index.md:7]: Hay una linea extra aqui :)
```suggestion
```

File: files/es/web/api/notifications_api/using_the_notifications_api/index.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+### Ejemplo
+
+En nuestro ejemplo de lista de tareas, incluimos un botón "Habilitar notificaciones" que, cuando se presiona, solicita permisos de notificación para la aplicación.
+
+```html
+<button id="enable">Habilitar notificaciones</button>
+```
+
+Al hacer clic, se llama a la función `askNotificationPermission()`:
```
Comment (Vallejoanderson) [inline files/es/web/api/notifications_api/using_the_notifications_api/index.md:69]:
```suggestion
Al hacer click, se llama a la función `askNotificationPermission()`:
```

File: files/es/web/api/notifications_api/using_the_notifications_api/index.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+    }
+  });
+});
+```
+
+### Resultado
+
+{{ EmbedLiveSample('Tag_example', '100%', 30) }}
+
+## Especificaciones:
```
Comment (Vallejoanderson) [inline files/es/web/api/notifications_api/using_the_notifications_api/index.md:258]:
```suggestion
## Especificaciones
```

File: files/es/web/api/notifications_api/using_the_notifications_api/index.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+
+{{Specifications("api.Notification")}}
+
+## Compatibilidad con navegadores
+
+{{Compat("api.Notification")}}
+
+## See also
```
Comment (Vallejoanderson) [inline files/es/web/api/notifications_api/using_the_notifications_api/index.md:266]:
```suggestion
## Véase también
```

**Review by Vallejoanderson — CHANGES_REQUESTED** [review]

File: files/es/web/api/notifications_api/using_the_notifications_api/index.md [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+
+Debido a los abusos de las notificaciones automáticas en el pasado, los navegadores web y los desarrolladores han comenzado a implementar estrategias para ayudar a mitigar este problema. Solo debe solicitar el consentimiento para mostrar notificaciones en respuesta a un gesto del usuario (por ejemplo, hacer clic en un botón). Esta no es solo la mejor práctica: no debe molestar a los usuarios con notificaciones que no aceptaron, sino que, en el futuro, los navegadores rechazarán explícitamente las solicitudes de permiso de notificación que no se activen en respuesta a un gesto del usuario. Firefox ya lo está haciendo desde la versión 72, por ejemplo, y Safari lo ha hecho durante algún tiempo.
+
+Además, en Chrome y Firefox no puede solicitar notificaciones a menos que el sitio sea un contexto seguro (es decir, HTTPS), y ya no puede permitir que se soliciten permisos de notificación desde {{htmlelement("iframe")}} de origen cruzado.
```
Comment (davbrito) [inline files/es/web/api/notifications_api/using_the_notifications_api/index.md:29]:
```suggestion
Además, en Chrome y Firefox no puede solicitar notificaciones a menos que el sitio sea un contexto seguro (es decir, HTTPS), y ya no puede permitir que se soliciten permisos de notificación desde un {{htmlelement("iframe")}} de origen cruzado.
```

**Review by davbrito — CHANGES_REQUESTED** [review]

**Review by Vallejoanderson — APPROVED** [review]

**Pull request merged**